{
    "title" : "There is a problem connecting to the Coveo Sitecore Content web service",
    "uri" : "http://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "printableUri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "clickUri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "uniqueId" : "42.4247$http://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "excerpt" : "I have a different issue. ... I already have Coveo collection for more than 10 websites all exists in same content tree but different urls. ... Now I am trying to add a new collection and on on reb...",
    "firstSentences" : null,
    "summary" : null,
    "flags" : "HasHtmlVersion",
    "hasHtmlVersion" : true,
    "hasMobileHtmlVersion" : false,
    "score" : 1135,
    "percentScore" : 75.807884,
    "rankingInfo" : null,
    "rating" : 3.0,
    "isTopResult" : false,
    "isRecommendation" : false,
    "titleHighlights" : [ ],
    "firstSentencesHighlights" : [ ],
    "excerptHighlights" : [ ],
    "printableUriHighlights" : [ ],
    "summaryHighlights" : [ ],
    "parentResult" : null,
    "childResults" : [ ],
    "totalNumberOfChildResults" : 0,
    "raw" : {
      "systitle" : "There is a problem connecting to the Coveo Sitecore Content web service",
      "sysauthor" : "vineethmaniyath",
      "sysurihash" : "j93EgUIv3CnaCif8",
      "urihash" : "j93EgUIv3CnaCif8",
      "numberofanswers" : 0,
      "sysuri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "sysprintableuri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "sitemappriority" : 0.1,
      "systransactionid" : 160358,
      "questionaskeddatestring" : "·\n            May 19, 2016 at 09:20 AM ·",
      "tags" : [ "coveoforsitecore" ],
      "sysconcepts" : "coveo ; sitecore ; websites ; h2 ; head ; setup ; wait ; guess ; extranet ; 2fcoveocontent ; 2fwebservice ; request",
      "concepts" : "coveo ; sitecore ; websites ; h2 ; head ; setup ; wait ; guess ; extranet ; 2fcoveocontent ; 2fwebservice ; request",
      "printableuri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "documenttype" : "WebPage",
      "sysindexeddate" : 1517942589000,
      "permanentid" : "0556cedf346966a647365cde2cc66f4cf591aa9c5ec6682de261cde7d416",
      "syslanguage" : [ "English" ],
      "transactionid" : 160358,
      "title" : "There is a problem connecting to the Coveo Sitecore Content web service",
      "date" : 1463616000000,
      "objecttype" : "Answers",
      "sitemapchangefrequency" : "Monthly",
      "hasanswer" : "false",
      "audience" : [ "Administrator", "Developer" ],
      "commentauthor" : [ "Jean-François L'Heureux \u2666\u2666" ],
      "hasacceptedanswer" : "false",
      "sourcetype" : "Sitemap",
      "sysconnectortype" : "SitemapCrawler",
      "rowid" : 5448213,
      "numberofcomments" : 1,
      "size" : 66071,
      "sysdocumenttype" : "WebPage",
      "questionvotes" : 0,
      "clickableuri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "syssource" : "AnswersCloud",
      "orderingid" : 212384744911578512,
      "acceptedanswers" : 0,
      "syssize" : 66071,
      "sysdate" : 1463616000000,
      "author" : "vineethmaniyath",
      "source" : "AnswersCloud",
      "questionbodyhtml" : "<div class=\"question-body\">\n                <p>Hi,</p> \n<p>I have a different issue. I already have Coveo collection for more than 10 websites all exists in same content tree but different urls.</p> \n<p>Now I am trying to add a new collection and on on rebuild I am getting an error </p> \n<pre><code> ERROR    08:03:59 AM    getcreative error: There is a problem connecting to the Coveo Sitecore Content web service (The request failed with the error message:--<html><head><title>Object moved</title></head><body><h2>Object moved to <a href=\"/404?item=%2fcoveo%2fwebservice%2fcoveocontent&user=extranet%5cAnonymous&site=modules_website\">here</a>.</h2></body></html>--.).\n\n</code></pre> \n<p>Since we have a different server for Coveo and we search it using a WCF service. </p> \n<p>Any guess what can be the error?</p> \n<p>Thanks</p> \n<p>Vineeth</p>\n                </div>",
      "sitemaplastmodified" : 1463616000000,
      "collection" : "default",
      "syssourcetype" : "Sitemap",
      "indexeddate" : 1517942589000,
      "questionnumber" : 8066,
      "connectortype" : "SitemapCrawler",
      "filetype" : "html",
      "sysclickableuri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "sysfiletype" : "html",
      "language" : [ "English" ],
      "sitemapparenturl" : "http://answers.coveo.com/questions/1.xml",
      "questionauthor" : "vineethmaniyath",
      "sitename" : "Answers",
      "sysrowid" : 5448213,
      "uri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
      "syscollection" : "default",
      "sitelanguage" : [ "English" ]
    },
    "Title" : "There is a problem connecting to the Coveo Sitecore Content web service",
    "Uri" : "http://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "PrintableUri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "ClickUri" : "https://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "UniqueId" : "42.4247$http://answers.coveo.com/questions/8066/there-is-a-problem-connecting-to-the-coveo-sitecor.html",
    "Excerpt" : "I have a different issue. ... I already have Coveo collection for more than 10 websites all exists in same content tree but different urls. ... Now I am trying to add a new collection and on on reb...",
    "FirstSentences" : null
  }